Makefile

```diff
@@ -410,9 +411,19 @@ test/cluster/cleanup:
 	./scripts/cleanup_test_cluster.sh
 .PHONY: test/cluster/cleanup
 
+# Target to autogenerate moq files
+moq/generate: PRE_DELETE_AUTOGENERATED_MOQ_FILES ?= true
+moq/generate: moq
+	@if [ "$(PRE_DELETE_AUTOGENERATED_MOQ_FILES)" = "true" ]; then \
+		echo "Pre-Deleting autogenerated moq files ..." ; \
```

Comment:
I believe we should use "@echo" instead of "echo"

Reply:
When using `@` in the `if` the body is already muted:
```
msoriano@localhost:~/go/src/github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager (revamp-moq-generation)$ make generate
Pre-Deleting autogenerated moq files ...
/home/msoriano/go/src/github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/workers/worker_interface_moq.go
...
Deletion of autogenerated moq files finished.
...
``